<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editprofile</class>
 <widget class="QDialog" name="editprofile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_up">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>1241</width>
     <height>571</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_cancelep">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>470</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fN">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_lN">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pN">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>220</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Phone Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ad">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>280</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ed">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>340</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Education</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_u2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>150</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_u1">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>90</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_u3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>210</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_u4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>270</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_u5">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>330</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>470</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Changes</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_u6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>390</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>400</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Company</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_update">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_jhlogo_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_jhlogo_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_update">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Profile</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_uppf">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
